src/snowflake/sqlalchemy/snowdialect.py

```diff
@@ -748,17 +764,12 @@ def get_view_definition(self, connection, view_name, schema=None, **kw):
 
     def get_temp_table_names(self, connection, schema=None, **kw):
         schema = schema or self.default_schema_name
-        if schema:
-            cursor = connection.execute(
-                text(
-                    f"SHOW /* sqlalchemy:get_temp_table_names */ TABLES \
-                    IN {self._denormalize_quote_join(schema)}"
-                )
-            )
-        else:
-            cursor = connection.execute(
-                text("SHOW /* sqlalchemy:get_temp_table_names */ TABLES")
+        cursor = connection.execute(
+            text(
+                f"SHOW /* sqlalchemy:get_temp_table_names */ TABLES \
+                IN {self._denormalize_quote_join(schema)}"
```

Comment:
Should we add the SCHEMA keyword in the filter? as in the line https://github.com/snowflakedb/snowflake-sqlalchemy/blob/a05a87ce8342e988ad0d390d86c3955f6bfdec46/src/snowflake/sqlalchemy/snowdialect.py#L694